agents/builtin/mysql/slowlog/slowlog.go

```diff
@@ -186,51 +198,50 @@ func (m *SlowLog) Run(ctx context.Context) {
 }
 
 // getSlowLogFilePath get path to  MySQL slow log and check correct config.
-func (m *SlowLog) getSlowLogFilePath() (string, error) {
+func (m *SlowLog) getSlowLogFilePath() (string, float64, error) {
 	var isSlowQueryLogON int
 	row := m.db.QueryRow("SELECT @@slow_query_log")
 	if err := row.Scan(&isSlowQueryLogON); err != nil {
-		return "", errors.Wrap(err, "cannon select @@slow_query_log")
+		m.l.Errorf("cannot select @@slow_query_log: %s", err)
 	}
 	if isSlowQueryLogON != 1 {
-		return "", errors.Errorf("cannot parse slowlog: @@slow_query_log is off: %v", isSlowQueryLogON)
+		m.l.Errorf("cannot parse slowlog: @@slow_query_log is off: %v", isSlowQueryLogON)
 	}
+
```

Comment:
There has to be a comment explaining why we log "cannot parse slowlog" message when it is disabled, but continue parsing it anyway. That will look like a missing `return` for anyone reading this code.

Reply:
Added.